src/core/namespace.c

```diff
@@ -182,14 +179,6 @@ static const MountEntry protect_system_yes_table[] = {
         { "/usr",                READONLY,     false },
         { "/boot",               READONLY,     true  },
         { "/efi",                READONLY,     true  },
-#if HAVE_SPLIT_USR
-        { "/lib",                READONLY,     true  },
-        { "/lib64",              READONLY,     true  },
-        { "/bin",                READONLY,     true  },
-#  if HAVE_SPLIT_BIN
```

Comment:
Did you intend to drop support for split-bin? I think there are some distros that have merged /usr but did not merge bin and sbin?

In any case, if support for split-bin is being dropped, there are some other places this would need to be reflected.

Reply:
Oh, never mind. I now see that this was nested inside HAVE_SPLIT_USR in the first place, and is unneeded if both bin and sbin reside under /usr.

Reply:
yeah, only Arch has done that, so this is not being dropped

Reply:
Do you happen to know why only arch did this, btw? Every time I boot up a debian live system I get annoyed that auto-completion doesn't find sbin binaries. Which is even sillier for any that can be run without privileges.

Reply:
In Gentoo, we've merged bin and sbin as well for mergedusr installs. I'd assumed that most people were doing this for merged-usr, given it feels natural someone would end up wanting to do it later anyway - so just combined the two.